packages/kas/package.json

```diff
@@ -23,19 +23,18 @@
     ],
     "scripts": {
         "prepublishOnly": "../../utils/package-pre-publish-check.sh",
-        "gen:parsers": "node src/parser-generator.ts",
-        "test": "bash -c 'yarn --silent --cwd \"../..\" test ${@:0} $($([[ ${@: -1} = -* ]] || [[ ${@: -1} = bash ]]) && echo $PWD)'"
```

Comment:
I removed this. As far as I know, nobody cd's into the various package directories and runs tests. It's just as easy to run from the root, and you can just as easily run `pnpm test packages/kas` from the root!